src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -149,11 +156,39 @@ button.NormalAnnouncementBubble--deleteMenuItem {
   white-space: nowrap;
 
   .tooltip-inner {
-    max-width: 30rem;
+    width: 18.75rem;
     padding: @size_2xs @size_s;
   }
 }
 
+.NormalAnnouncementBubble--readReceipts--icon {
+  .pathColor(@neutral_medium_gray);
+  .margin--right--2xs();
+  width: 0.865rem;
+  height: 0.663rem;
```

Comment:
How come these are so specific?

Reply:
These dimensions were specified in the Figma, so I tried to follow them as accurately as I could